src/main/java/edu/cornell/kfs/kim/service/impl/CuUiDocumentServiceImpl.java

```diff
@@ -11,9 +11,12 @@
 import org.kuali.kfs.kim.impl.role.Role;
 import org.kuali.kfs.kim.impl.role.RoleMember;
 import org.kuali.kfs.kim.service.impl.UiDocumentServiceImpl;
+import org.kuali.kfs.krad.service.BusinessObjectService;
 
 public class CuUiDocumentServiceImpl extends UiDocumentServiceImpl {
 
+    private BusinessObjectService businessObjectService;
```

Comment:
Does KualiCo still want to know when we have to add to overlays to increase visibilities of things?  If so, we should let KualiCo know UiDocumentService's businessObjectService should be protected

Reply:
Good point, I'll add a note about this in the report-findings-to-KualiCo sub-task.